SergiiDotsenko/src/library/Reader.java

```diff
@@ -26,14 +26,20 @@ public void returnBook() {
     public void printFullInfo() {
         System.out.println(String.format("reader id is %d, books on hands are %d", readerID, booksOnHands));
```

Comment:
if you want to make method void so use System.out.printf (it's more common), but it would be better make method return String